.circleci/scripts/calculate_directory_hash.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -eou pipefail
+
+HASH_DIR=${HASH_DIR:-$(pwd)}
+
+# We use sort -k 2 here to sort by the file name
+git ls-files :${HASH_DIR} \
```

Comment:
Annoyingly, `find . | sort` gives different results based on OS, so we're just going to use something that's at least consistent here.